server/models/Request.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const timeSlot = new Schema({ start: Date, end: Date });
+
+const requestSchema = new Schema({
+  ownerId: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  sitterId: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  duration: {
+    type: timeSlot,
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    default: null,
+  },
+  totalCost: {
+    type: Number,
+    default: 0,
+  },
+  paid: {
+    type: Boolean,
+    default: false,
+  },
+  completed: {
+    type: Boolean,
+    default: false,
+  },
+  rating: {
```

Comment:
We will be designing a separate model for ratings/reviews (to rate a sitter after a job has been completed). I don't think we need this field here :)

Reply:
Removed